src/components/global/Modal.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import ReactDOM from 'react-dom';
+
+const modalRoot = document.querySelector('#modal-root');
+
+export default function Modal({ dismissOnClickOutside, onCancel, children, show }) {
+    if(!show){
+        return null;
+    }
+
+    return ReactDOM.createPortal(
+        <div className="modalContainerBg" aria-hidden="true" onClick={e => {
+            if (dismissOnClickOutside && e.target.parentElement === modalRoot) {
+                onCancel();
+            }
+        }}>
+            <div className="modalContainer">
+                {children}
+            </div>
+        </div>
```

Comment:
There isn't anything wrapping around the html elements, like a set of brackets or something. Just wanted to make sure this is okay?

Reply:
It wrapped with the brackets from ReactDOM.createPortal(....html elemets...)

Reply:
No, there are 2 arguments to the createPortal function. The first is the html elements. The second is ```modalRoot```, which is where the portal is created. You haven't wrapped the html elements in brackets

Reply:
oh, that's how it was done in one of the 750 labs which i looked at when making the modal plus when i tried to put it in brackets it would throw errors so i left it

Reply:
okay cool, if it works then its good

Reply:
I can prove that's true. ^ Should be good